config/_default/menus.toml

```diff
@@ -65,5 +65,5 @@
 
 [[main_right]]
   name = 'support'
-  url = "https://docs.2i2c.org/en/latest/admin/howto/support.html"
+  url = "https://2i2c.freshdesk.com/"
```

Comment:
Maybe this one instead? https://docs.2i2c.org/en/latest/support.html
@choldgraf do you have any thoughts on this one?

Reply:
ahh that's a good point @damianavila - that page has more context, agreed we should use that one instead